### PR TITLE
New 404 .svg

### PR DESCRIPTION
**Pull request by grngxd**
This pull request changes the 404 image to a more scratch-themed 404, with a white and yellow colorway and simplistic but bold text.

### CODE REVIEW

File: config.xml [hunk 1]
```diff
@@ -1,59 +1,59 @@
-<?xml version='1.0' encoding='utf-8'?>
```
Comment (locness3) [inline config.xml:1]: What changed here?

File: asset/404-img.svg [hunk 2]
```diff
@@ -1 +1,3 @@
...
...ybNDDlWDyQBFdgQC7qpZKpeta2zMgY0dNAwWU2SM9m9wouxqUy2XyKmjO2OTv6lKppCLv4sWztEtzTwE3zZOHpjvxfz8k8v56vYMtI/q3JJhLYCDRUSglRfR+RqlU2lcul9H/+DUAuILdqjd6DX1QGMyAkXxJJmB7gdoQ+rFwjoKF5zRthy5f59mVg7+kd3pHkEaYSWF+vg6nyiacOlUNasnxQgmmFiiiKyxBqVTCZoN7yuXy1Ri519rPyiukSDsJGnXnpg/EI6hQENkkes8MG+7Nw400yEGFgicP5FMHAOBTlmnvThn6ud3OCK0tWAcUo4rxb9SwSVlQywGTduw2STSoRq0E7mfusW4KjEP7xTUabkI5Rik3OihE0IL9Oy9eLzSysx8EEyiiKywH1BKo9vR5jNxblt6Yk0ZK+LgWYZYIywSViKaDbsNKVwhh+HiI6gxBgRuZG6iAVpAHaOH5Hcyh/PG/vzk9WMqMYQGAZoqB7fIQi5TIAjCxTJZFO3E7bl4l40uSQfjGuV8EEyiiKywHqi3cUC6X9wHAFwDgWt5CuZqMvUSLYWWE2Ks7ThdYjrNYzqhZey/ewItnqEB6MM6LKnQOy3YePDlbu/bkrAxJCKGAUZ7Cg2n6STA1oYiu0AqqPWDe080AgOOqoIvEhEsMmk63QIF0s4q26yk86LdJ6hHs3XnxeuEBNX0pmBTRFdqBCqiry+XyZgC4npr4ejXq8SEabdfXGtIXb/rhkn9ftrtrS71MwE3ELvoM9irQCnHPxGNHhU9fa40A6hOiA3V69zrRI0mCfOThZ4Sc5/s/eETIeRLGahq9d503ik9ivEaZGo6e8iH9/OWXCTlPH/EooDTstRtaoLwpMg29XzUmoAt3qSK6gg+OeRj9ZiqcrpZMSB2k2hGORyWYj4I4nNcjm+gF+vw9SF8jteT0s2ACRXSFDvEq1VT30a/toQJqT8wbHPU8rCzspbU/2fO2KoK7X6Bjmv77GP030PfmPX+zY+fp62txb4j6XTCBIrpCCDxIxw3U5LeLPkM76BCF16hWdJA+q+p5WHm4x2Ml2UH51a5l+IWXz8x7eApQvgOe47zf7SmsBMEEiugKAoD0v58Ohh3U/LeZbnjYKwumYK/e5+UYFUSv0tf/8DxbCgEgykcqMdgmZWUCAP4/FPuqiCpgPLkAAAAASUVORK5CYII="/>
```
Comment (locness3) [inline asset/404-img.svg:2]: A PNG in an SVG? what's the point?

Comment (micahlt) [reply asset/404-img.svg:2]: @AyzansAlt Yeah, I don't get the point here.

Comment (locness3) [reply asset/404-img.svg:2]: Prolly because the code refers the SVG and @AyzansAlt didn't want to change that?

Comment (AyzansAlt) [reply asset/404-img.svg:2]: Yeah, sorry for not responding earlier, I'm kinda leaving scratch and github for a white, and @locness3  is right

Comment (AyzansAlt) [reply asset/404-img.svg:2]: Basically all i know is raster art :/

Comment (locness3) [reply asset/404-img.svg:2]: That's okay! We'd just need to update the code a little (you can even do it!)

Comment (AyzansAlt) [reply asset/404-img.svg:2]: yAy, I just thought that you wanted SVG art cause all I see is SVG art in all of @micahlt's projects

**Comment by AyzansAlt** [comment]
Just asking, where is the 404 file because i can't seem to find it?

**Comment by locness3** [comment]
it's not found-

**Comment by AyzansAlt** [comment]
should i just merge this because i dont know what to do with the nonexistent 404.js

**Comment by micahlt** [comment]
> should i just merge this because i dont know what to do with the nonexistent 404.js @locness3 @micahlt

@AyzansAlt Do **not** merge anything that has errors!

**Comment by grngxd** [comment]
> > should i just merge this because i dont know what to do with the nonexistent 404.js @locness3 @micahlt
> 
> @AyzansAlt Do **not** merge anything that has errors!

what errors are in here exactly?

**Comment by grngxd** [comment]
@micahlt

**Pull request closed without merging**